jupyterhub/files/hub/jupyterhub_config.py

```diff
@@ -420,14 +420,18 @@ def camelCaseify(s):
         exec(compile(source=file_content, filename=file_name, mode="exec"))
 
 # load potentially seeded secrets
-c.JupyterHub.proxy_auth_token = get_secret_value("JupyterHub.proxy_auth_token")
+c.ConfigurableHTTPProxy.auth_token = get_secret_value(
+    "ConfigurableHTTPProxy.auth_token"
+)
 c.JupyterHub.cookie_secret = a2b_hex(get_secret_value("JupyterHub.cookie_secret"))
 c.CryptKeeper.keys = get_secret_value("CryptKeeper.keys").split(";")
 
 # load hub.config values, except potentially seeded secrets
 for section, sub_cfg in get_config("hub.config", {}).items():
     if section == "JupyterHub" and sub_cfg in ["proxy_auth_token", "cookie_secret"]:
```

Comment:
What does this disallowed list do? To make sense, sub_config must always be a dict, so these `sub_cfg in [list]` conditions will always be False. Or am I misunderstanding something?

Reply:
Ah yes this logic is just malfunctional because I did the mental mistake of thinking sub_cfg was string representing the sub config key name (`"proxy_auth_token"`).

Resolved by fcea63f I think.